src/rtk/features/singleRights/singleRightsSlice.ts

```diff
@@ -265,7 +266,6 @@ const singleRightSlice = createSlice({
           if (sws.service?.identifier === serviceID) {
             sws.isLoading = false;
             sws.status = ServiceStatus.Error;
-            sws.errorCode = 'HTTPError';
```

Comment:
I think this is the source of the bug with the missing alert.
ErrorCode is no longer part of the sate, then this code needs to be set some other way

Reply:
Fixed